src/components/Buildbaron/BuildBaronContent.tsx

```diff
@@ -31,6 +34,15 @@ const BuildBaronCore: React.FC<BuildBaronCoreProps> = ({
     dispatchBanner.clearAllBanners();
   }, [tab]); // eslint-disable-line react-hooks/exhaustive-deps
   const [createdTicketsCount, setCreatedTicketsCount] = useState<number>(0);
+
+  const { data } = useQuery<GetSpruceConfigQuery>(GET_SPRUCE_CONFIG);
+  const spruceConfig = data?.spruceConfig;
+  const jiraHost = spruceConfig?.jira?.host;
+
+  const jiraSearchString = eventData?.searchReturnInfo?.search;
+  const jqlEscaped = encodeURIComponent(jiraSearchString);
+  const jiraSearchLink = `https://${jiraHost}/secure/IssueNavigator.jspa?jql=${jqlEscaped}`;
```

Comment:
this should probably live in `constants/externalResources.ts`  https://github.com/evergreen-ci/spruce/blob/c305da71f8ef2c0aca13382efb70c3c0220c4654/src/constants/externalResources.ts#L4

Reply:
It's not really a constant, since it includes the jiraHost and the jira search string. I can technically move this part over: /secure/IssueNavigator.jspa?jql=$ , but I don't think I'd gain enough and it would probably feel clunky. This is not a very strong opinion though so if you prefer, I can move it over.

Reply:
nvm, I'll model it after your getIdeUrl.